flow-core/src/utils/logger/logger-types.ts

```
@@ -146,48 +151,87 @@ export interface StructuredLogger {
+export type LogLevel = 'debug' | 'info' | 'warn' | 'error' | 'critical';
+
+/**
+ * Valid log levels array for validation and iteration
+ */
+export const validLogLevels = ['debug', 'info', 'warn', 'error', 'critical'] as const;
+
+/**
+ * Log level numeric values for comparison and filtering
+ */
+export const LogLevelValues = {
+  debug: 0,
```

ESLint

[error] Unquoted property 'debug' found.

(quote-props)